gg-pac-team3-front/src/components/layout/Nav.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import logo from '../../assets/logos/blue.png';
+import styles from './nav.module.css';
+import { NavLink, Outlet} from "react-router-dom";
+
+function Nav(props){
```

Comment:
Use arrow function syntax when possible, i.e.

`const Nav = (props) => {`

`const` functions have a couple of benefits over old school function declarations, mainly that they are immutable and avoid wierdness with the `this` keyword.

Reply:
Switched to arrow functions!